main.go

```diff
@@ -99,10 +101,49 @@ func run(p params) error {
 		return errors.Wrap(err, "could not find failed tests")
 	}
 
-	err = j.createIssuesOrComments(failedTests)
+	issues, err := j.createIssuesOrComments(failedTests)
 	if err != nil {
 		return errors.Wrap(err, "could not create issues or comments")
 	}
+	return j.createHtml(issues)
+}
+
+//go:embed htmlOutput.html.tpl
+var htmlOutputTemplate string
+
+func (j junit2jira) createHtml(issues []*jira.Issue) error {
+	if j.htmlOutput == "" || len(issues) == 0 {
+		return nil
+	}
+	out := os.Stdout
+	if j.htmlOutput != "-" {
+		file, err := os.Create(j.htmlOutput)
+		if err != nil {
+			return fmt.Errorf("could not create file %s: %w", j.htmlOutput, err)
+		}
+		out = file
+		defer file.Close()
+	}
+	return j.renderHtml(issues, out)
+}
+
+type htmlData struct {
+	Issues  []*jira.Issue
+	JiraUrl string
+}
+
+func (j junit2jira) renderHtml(issues []*jira.Issue, out io.Writer) error {
+	t, err := template.New(j.htmlOutput).Parse(htmlOutputTemplate)
```

Comment:
Shouldn't we be using `html/template` rather than `text/template`?

Reply:
OK, I switched to `html` although I needed to change template and remove `<h4>` from `<title>` as it was escaped. This is only styling change that will change the size of the text in prow but I think we should do that same with other artifacts.
![image](https://github.com/stackrox/junit2jira/assets/1616386/4753c064-1370-4de4-ad8c-47a4fda037e7)

Reply:
Hm, I'm afraid some of these (like cluster-related stuff in case of clusters from the pool?) might be added by code not under our control.

Reply:
We add them all https://github.com/search?q=repo%3Astackrox%2Fstackrox%20%3Ch4%3E&type=code

Reply:
I meant entries like these [here](https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/branch-ci-stackrox-stackrox-nightlies-ocp-4-10-qa-ebpf-e2e-tests/1688779070789128192).

But it seems they already use a smaller font.
![Screenshot from 2023-08-08 12-39-02](https://github.com/stackrox/junit2jira/assets/489420/a6729e59-3da1-46b8-98ef-a6783f77074d)